Scripts/Taxon/__init__.py

```diff
@@ -77,20 +77,13 @@ def set_code(self):
 			
 	def set_blast(self,fileName, PathtoBlastFiles):	
 		print(fileName[:10] + ': Parsing blast report')
-		test = ''
 		for blastFileName in os.listdir(PathtoBlastFiles):
 			if re.search(fileName[:10], blastFileName):
 				blastrecords = NCBIXML.parse(open(PathtoBlastFiles + blastFileName,'r'))		
 				for blast in blastrecords:
 					if blast.descriptions:
 						self.blast.append(blast)
 		
-		if test == '':
-			outfile = open('log_' + self.code,'a')
-			outfile.write('no BLAST file found for ' + self.code)
-			outfile.close()
```

Comment:
This was part of a method from the first version of the pipeline. In this method, if there was not an XML for certain sps, then the pipeline would do the blast analysis to produce it. I removed the method some time ago. This piece was just a remnant.